modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/topology/PartitionsEvictManager.java

```diff
@@ -60,6 +61,8 @@
     private static final int DEFAULT_SHOW_EVICTION_PROGRESS_FREQ_MS = 2 * 60 * 1000;
 
     /** Eviction progress frequency property name. */
+    @SystemProperty(value = "Eviction progress frequency in milliseconds", type = Long.class,
+        defaults = "" + DEFAULT_SHOW_EVICTION_PROGRESS_FREQ_MS)
     private static final String SHOW_EVICTION_PROGRESS_FREQ = "SHOW_EVICTION_PROGRESS_FREQ";
```

Comment:
Should we consider making this fields `public` too?

Reply:
Yes. Fixed.